src/supermarket/app/controllers/concerns/collaborator_processing.rb

```diff
@@ -37,7 +37,20 @@ def add_group_members_as_collaborators(resource, group_ids)
 
   def remove_collaborator(collaborator)
     authorize!(collaborator, "destroy?")
+    cookbook_related = nil
+    if collaborator.resourceable_type == "Cookbook"
```

Comment:
use either small case or capital 
"cookbook" or "COOKBOOK"

Reply:
Actually this is how we are storing resourceable_type in collaborator table .. but yea i can make it a constant in model and refer here